documentation/using/master.adoc

```diff
@@ -2,6 +2,7 @@ include::shared/version-dependent-attrs.adoc[]
 include::shared/attributes.adoc[]
 
 :context: str
+:BookURLDeploying: https://strimzi.io/docs/latest/
```

Comment:
We really need to figure this out. This URL does not work:
* It is just for the latest docs, but you need to have the link working lso in the versioned docs.
* We have also the docs in the download archive
* There are also discussions about offline docs in the OperatorHub to go with the offline operators. So we need either some relative links or no links I guess.

Reply:
Tricky. I guess we can only use absolute links between guides, as we have _overview_, _quickstart_ etc in the path. The path could be split with more variables to represent {BookName}, {ProductVersion}, but we'd still need to make sure this was properly represented when branching. Not sure how that might translate to OperatorHub doc.

I suppose we could use relative links if we created one meta book upstream with all the other books sitting underneath. 

The simplest option is for the guide-to- guide links to go. Shame to lose them though.

Reply:
For upstream, I think we can make it work with just using relative links ... e.g. `using.html` and `deploying.html` etc. We would need to do some changes to the website I guess, but it should be possible and it might work. It should work in the downloads as well. Not sure how the OperatorHub would work ... we might need to talke that when/if it happens.